src/minecraft/commands/WeightCommand.js

```diff
@@ -0,0 +1,70 @@
+const MinecraftCommand = require('../../contracts/MinecraftCommand')
+const fetch = require('node-fetch')
+
+class WeightCommand extends MinecraftCommand {
+  constructor(minecraft) {
+    super(minecraft)
+
+    this.name = 'weight'
+    this.aliases = ['we']
+    this.description = "Tells a player his weight"
+  }
+
+  async onCommand(username, message) {
+    let args = message.split(' ')
+    let ign = args[1]
+
+    if (args.length == 1) {
+        ign = `${username}`
+    }
+
+    fetch(`https://api.mojang.com/users/profiles/minecraft/${ign}`)
+      .then(res => {
+        if (res.status != 200) {
+        return this.send(`/gc No MC account found for ${ign}`)
+      }
+    })
+
+    ign = await getTrueIgn(ign);
+
+    const apiData = await getApiData(ign);
+
+    if (apiData.status != 200) {
+      this.send(`/gc ${apiData.reason}`)
+    }
+
+    if (apiData.data.skills.apiEnabled == false) this.send(`/gc This user has skill API disabled`)
```

Comment:
These aren't needed as disabling skills API will just show 0 weight for it, the total weight and overflow are already calculated in the `data.weight` and `data.weight_overflow` fields of the response.

Reply:
Removed that